plugins/net_plugin/net_plugin.cpp

```diff
@@ -519,14 +519,15 @@ namespace eosio {
     uint32_t            sync_last_requested_num;
     uint32_t            sync_req_span;
     connection_ptr      source;
+    bool                active;
```

Comment:
Need to initialize in the constructor or better yet initialize right here.

Reply:
got it.